### PR TITLE
[CWG Motion 6 2024-06] P2963R3 Ordering of constraints involving fold expressions

### LINKED ISSUE #7082: [2024-06 CWG Motion 6] P2963R3 Ordering of constraints involving fold expressions
https://wiki.edg.com/pub/Wg21stlouis2024/StrawPolls/P2963R3.pdf

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #7082
Also fixes cplusplus/papers#1638

### CODE REVIEW

File: source/templates.tex [hunk 1]
```diff
@@ -1719,6 +1720,47 @@
...
+Let $N$ be the number of elements
+in the pack expansion parameters\iref{temp.variadic}.
+
+\pnum
+A fold expanded constraint whose \grammarterm{fold-operator} is \tcode{\&\&}
+is satisfied if it is a valid pack expansion and
+if $N = 0$ or if for each $i$ where $0 \le i < N$ in increasing order,
+$C$ is satisfied
+when replacing each pack expansion parameter
+with the corresponding $i$-th element.
```
Comment (burblebee) [inline source/templates.tex:1739]: we typically use the syntax `$i^\text{th}$` for this

File: source/templates.tex [hunk 2]
```diff
@@ -1719,6 +1720,47 @@
...
+the smallest $i$ for which the constraint is satisfied.
+
+\pnum
+\begin{note}
+If the pack expansion expands packs of different size,
+then it is invalid and the fold expanded constraint is not satisfied.
+\end{note}
+
+\pnum
+Two fold expanded constraints are \defn{compatible for subsumption}
```
Comment (burblebee) [inline source/templates.tex:1760]: also add an index under subsumption `\indextext(subsumption!compatible for}`?

Comment (jensmaurer) [reply source/templates.tex]: fixed

File: source/templates.tex (hunk 1 above)
Comment (jensmaurer) [reply source/templates.tex]: fixed

**Review by burblebee — APPROVED** [review]

**Pull request merged**